pom.xml

```diff
@@ -77,12 +77,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-step-api</artifactId>
-            <version>2.13</version>
+            <version>2.16</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.30</version>
+            <version>2.31-rc852.4e8faa4117f6</version> <!-- TODO https://github.com/jenkinsci/workflow-api-plugin/pull/81 -->
```

Comment:
Hmm, this is what I have locally, but the tests still don't pass for me (not even intermittently). Maybe a subtle difference in IO behavior on macOS, or my machine is just too slow or something.

Reply:
I will look into it.

Reply:
I got a flake: one failure, one pass. Possibly something not fully honoring `flush`. Investigating.

Reply:
`TaskListenerDecoratorTest#smokes` is passing consistently for me after fa173a6.